### DISCUSSION TITLE
Renaming the ID kwargs for different command types in `tanjun.Client`, etc.

### DISCUSSION THREAD

**Discussion by googolgenius** [FasterSpeeding/Tanjun#361] [Ideas]
https://github.com/FasterSpeeding/Tanjun/blob/master/tanjun/clients.py#L552-L554

I personally feel like some of these kwargs here should be renamed to be more verbose and specific as especially `message_ids` and `user_ids` seem very confusing. They may refer to `hikari.User` ids or `hikari.Message` ids, etc if not looking at the type hint. 

Perhaps `message_cmd_ids` or `message_command_ids` would be a better naming scheme. For `command_ids` it could be deprecated to `slash_cmd_ids` or `slash_command_ids` or whatever you feel would work best, just specifying that it is for slash commands. 

As mentioned previously, it would be ideal in my opinion to rename and deprecate these kwargs and potentially other namings like this that apply to the library.

**Comment by FasterSpeeding** [comment]
hmmmmmmmmmmmmmmmmmmmm

**Comment by FasterSpeeding** [comment]
Yeah i'll add those aliases
